script_vts_1.py

```diff
@@ -0,0 +1,62 @@
+"""import of modules"""
+
+import argparse
+import os
+import sys
+from pathlib import Path
+
+if __name__ == '__main__':
+
+    # Create the parser to have arguments in the script
+    parser = argparse.ArgumentParser()
+
+    # Adding SHA argument (let required True for now?)
+    parser.add_argument('--SHA', type=str, required=True)
+
+    # Parse the argument
+    args = parser.parse_args()
+
+    #  TESTING CODE OR NO ? (might be good for the user to see the HASH?)
+    print("[+] Your SHA is: " + args.SHA)
+
+    # Sys.exit is good for handling error ? While loop until good result might be an option too...
+    # Might recode or put a helper function just before to list path when hitting tab...
+    def get_path(prompt):
+        """Get Path with input(), check if both path/file exist"""
+
+        try:
+            res = input("Please input an absolute path: " + prompt)
+            if os.path.exists(res):
+                print("[+] The path is valid!")
+                res = Path(res)
+
+                if res.is_file():
+                    print("[+] We did find the file!")
+                else:
+                    print("[-] We didn't found the file!")
+                    sys.exit(1)
+            else:
+                print("[-] The path is not valid!")
+                sys.exit(1)
+        except ValueError:
+            sys.exit(1)
```

Comment:
You should be able to exit same as in the HelloWorld.py file. Saves you needing to import the sys library.
```exit(1) #Process exited with a failure```
Glad to see you exitted with an error status. :)

Reply:
Well, I know I'm always doing like you're doing, just calling exit(1) or 0 if successful, but my linter said no you have to be more precise you need to import sys and write sys.exit(1) to really tell that you're using this function from this module. (This practice is not really python related in itself it's also common for other languages to impose this when you learn c++ they're telling you never ever do idk for example ```using namespace std``` to simply call cin or cout you should always precise like std:::cout to sya that yeah you want cout function from std) So technically they're right it's a better practice to do it like this. But if you want, I have two solutions, first one I totally follow what you said only put exit no sys module OR I can limit the importation by doing something like from sys import exit or something like this?

Reply:
@v3r1t4s My way is more secure. As we discussed, we're probably not going to be writing code that needs more security so in this case I'd go with the linter. Also a #TIL moment for me too.